pkg/function/delete_csi_snapshot_content_test.go

```diff
@@ -83,10 +83,13 @@ func (testSuite *DeleteCSISnapshotContentTestSuite) TestDeleteCSISnapshotContent
 			VolumeSnapshotClassName: snapshotClassName,
 		}
 		err = fakeSnapshotter.CreateContentFromSource(ctx, source,
-			snapshotContentName,
-			snapshotName,
-			snapshotNamespace,
-			deletionPolicy, nil)
+			deletionPolicy, metav1.ObjectMeta{
+				Name:      snapshotName,
+				Namespace: snapshotNamespace,
+			},
+			metav1.ObjectMeta{
+				Name: snapshotContentName,
+			})
```

Comment:
same here, let's create `snapshotMeta` and `contentMeta` before this line and pass that. It would make sure that reader understands what exactly is being passed.